README.md

```diff
@@ -88,14 +88,11 @@ More process details to follow, it's currently being worked out.
 
 <summary>Bastion configuration</summary>
 
-Update the user line to your email address
+Run 'az login'
 
-```shell
-Host *.platform.hmcts.net
-  User <your-email>@hmcts.net # must be lower case
-  PubkeyAuthentication no
-  ForwardAgent yes
-```
+Ensure ssh extension for the Azure CLI is installed: 'az extension add --name ssh'
+
+Run 'az ssh config --ip \* --file ~/.ssh/config'
```

Comment:
we should not be modifying all values, we've already had people hit issues because of this

```suggestion
Run `az ssh config --ip \*.platform.hmcts.net --file ~/.ssh/config`
```